src/atoms/Link/Link.native.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import styled, { ThemeContext } from 'styled-components/native';
+
+const StyledTouchable = styled.TouchableOpacity`
+  align-self: flex-start;
+`;
+
+const resolveSizeToTextProps = (linkSize) => {
+  //TODO: Add lineHeight prop for all cases
+  switch (linkSize) {
+    case 's':
+      return { size: 'xs', weight: 'regular' };
+    case 'm':
+      return { size: 'm', weight: 'regular' };
+    case 'l':
+      return { size: 'l', weight: 'regular' };
+    default:
+      return { size: 'xs', weight: 'regular' };
+  }
+};
+
+const Link = ({ children, visited, onClick, size }) => {
+  const theme = useContext(ThemeContext);
+  const [active, setActive] = useState(false);
+
+  const onPressIn = useCallback(() => {
+    setActive(true);
+  }, [setActive]);
+  const onPressOut = useCallback(() => {
+    setActive(false);
+  }, [setActive]);
+
+  const inactiveTextColor = visited ? theme.colors.secondary[900] : theme.colors.primary[800];
+  const activeTextColor = visited ? theme.colors.secondary[600] : theme.colors.primary[600];
+
+  return (
+    <StyledTouchable
+      activeOpacity={1}
+      onPress={onClick}
+      onPressIn={onPressIn}
+      onPressOut={onPressOut}
+      testID="ds-link-button"
+    >
```

Comment:
We will be passing `testId` from the app. That `testId` will be used in the e2e tests. We need to pass that `testId` to the root native element.